app/modules/individuals/resources.py

```diff
@@ -132,11 +258,20 @@ def patch(self, args, individual):
     @api.response(code=HTTPStatus.NO_CONTENT)
     def delete(self, individual):
         """
-        Delete a Individual by ID.
+        Delete an Individual by ID.
         """
-        context = api.commit_or_abort(
-            db.session, default_error_message='Failed to delete the Individual.'
-        )
-        with context:
-            db.session.delete(individual)
+        response = individual.delete_from_edm()
+        response_data = None
+        if response.ok:
+            response_data = response.json()
+            individual.delete()
```

Comment:
Does EDM return OK if the individual wasn't present anyway?

Reply:
Nope, it will return a 404 and success=False. We should delete from Houston even in this case, you are right.